packages/ap-storage/alliance_platform/storage/async_uploads/models.py

```diff
@@ -256,6 +256,17 @@ class AsyncFileMixin(AsyncFileMixinProtocol):
     they can download the file). This view will then redirect to the URL provided by
     :meth:`~alliance_platform.storage.async_uploads.storage.base.AsyncUploadStorage.generate_download_url`.
 
+    You can specify ``download_params`` on fields to control what arguments are passed through to :meth:`~alliance_platform.storage.async_uploads.storage.base.AsyncUploadStorage.generate_download_url`. For example::
+
+    expire_in_seconds = 60 * 60 * 6
+    download_params = {
+        "expire": expire_in_seconds,
+        "parameters": {"cache_control": f"public,max-age={expire_in_seconds}"},
+    }
+
+    # pass to a field on the model
+    image_file = AsyncFileField(download_params=download_params)
+
```

Comment:
not rendering properly, the code section needs to be indented. you can see at https://alliance-platform--107.org.readthedocs.build/projects/storage/107/api.html#fields

Reply:
Fixed @davecoates